core/data_migration_scripts/4_1_0_stages/EE_DMS_4_1_0_org_options.dmsstage.php

```diff
@@ -256,9 +256,6 @@ private function _handle_org_option($option_name, $value)
             case 'use_event_timezones':
                 $c->admin->use_event_timezones = ($value == 'Y');
                 break;
-            case 'full_logging':
-                $c->admin->use_full_logging = ($value == 'Y');
-                break;
```

Comment:
hmmmmm... not sure if it's kosher to remove stuff from older DMS's 

@mnelson4 
do you know if removing this `3.1 => EE4` config conversion from `EE_DMS_4_1_0_org_options` will mess anything up?

I have the mildest of concerns that it might have the potential to cause errors during migrations if anything happens to reference that option and it's not there, although I'm also skeptical of that actually happening. I also think it won't be an issue at all to simply leave this in place.

Reply:
@mnelson4 my point here is that $use_full_logging was removed - so perhaps would cause problem if tried to be accessed even during a migration.

All the $use_full_logging uses were removed as well - together with the class' property.

Reply:
Brent's right that we usually leave the older data migration scripts alone, seeing how they're not supposed to be bringing older database tables' data into conformity with the newest database schema, but the database tables that existed at the time.
BUT the config object is an exception because it's as soon as EE is installed, they're using the current class (not the class that existed during EE 4.1).
So in this case, seeing how the property has been removed from the `EE_Admin_Config` class, if we didn't remove this line, we'd get a warning during the migration (because that property wouldn't be defined).
So it turns out, where the data migration scripts are using config classes, like in this case, they actually DO need to be updated when the config classes get changed.
So this is good.